app/waf/src/models/pool.model.ts

```diff
@@ -59,6 +67,11 @@ export class Pool extends CommonEntity {
       response: true,
       example: 1,
     },
+    openapi: {
+      type: 'integer',
+      minimal: '1',
+      maxmium: '10',
```

Comment:
63

Reply:
it can be 'all', but I think we can use a special value in db to indicate 'all' and translate it to 'all' when generating AS3 declaration.